### PR TITLE
NETOBSERV-1443 Console plugin loki timeout should be configurable

### PR DESCRIPTION
**Pull request by jpinsonneau**
## Description

This PR expose `readTimeout` for console plugin. By default, the value is set to `30s`.

Here is an example with `5s`:

image
image

And `5m`:

image

## Dependencies

Based on https://github.com/netobserv/network-observability-operator/pull/467, merge it first !
https://github.com/netobserv/network-observability-console-plugin/pull/446

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [X] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [X] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

**Comment by jpinsonneau** [comment]
Rebased with changes:
- Renamed existing `loki.timeout` to `loki.writeTimeout` for processor
- Moved console plugin timeout into `loki.readTimeout`
- updated sample

See https://github.com/netobserv/network-observability-operator/pull/522/commits/f495691693ddc343552f3ccdc0d09493fb982dd4

File: config/samples/flows_v1beta2_flowcollector.yaml [hunk 1]
```diff
@@ -89,6 +89,8 @@ spec:
       name: loki
       # Change loki operator instance namespace
       # namespace: loki-operator
+    # Console plugin read timeout
+    readTimeout: 10s
```
Comment (jotak) [inline config/samples/flows_v1beta2_flowcollector.yaml:93]: it should be 30s here

**Comment by jotak** [comment]
There's a wrong value in the sample file, other than that lgtm

File: config/samples/flows_v1beta2_flowcollector.yaml (hunk 1 above)
Comment (jpinsonneau) [reply config/samples/flows_v1beta2_flowcollector.yaml:93]:
```suggestion
    readTimeout: 30s
```

Comment (jpinsonneau) [reply config/samples/flows_v1beta2_flowcollector.yaml]: Here we go, thanks !

**Comment by jpinsonneau** [comment]
Rebased without changes

**Pull request merged**